zjit/src/stats.rs

```diff
@@ -387,6 +410,8 @@ pub fn send_fallback_counter(reason: crate::hir::SendFallbackReason) -> Counter
         SendWithoutBlockDirectTooManyArgs         => send_fallback_send_without_block_direct_too_many_args,
         SendPolymorphic                           => send_fallback_send_polymorphic,
         SendNoProfiles                            => send_fallback_send_no_profiles,
+        FancyFeatureUse                           => send_fallback_fancy_call_feature,
```

Comment:
Call we call them `SendWithoutBlockIseqFancyFeatureUse => send_fallback_send_without_block_iseq_fancy_feature_use`?

I get that the `fancy_use_` counters are not limited to ISEQ calls, but the `FancyFeatureUse` side is used only for ISEQ calls right now. Until we apply it to other method types, it's useful information to have when we look at it.

Reply:
Actually, I noticed it has bmethod as well. Then maybe `SendWithoutBlockFancyFeatureUse => send_fallback_send_without_block_fancy_feature_use`, until we start using it for `send`?